core/templates/dev/head/editor/EditorServices.js

```diff
@@ -932,6 +932,15 @@ oppia.factory('statePropertyService', [
   };
 }]);
 
+// A data service that stores the current list of state parameter changes.
+// TODO(sll): Add validation.
```

Comment:
I'll add a note for us to discuss that in the frontend document. I'm thinking the TODO can be removed, though the validation function might be better placed in the stateParamChangesService.